Add parserManager and first version Resource Manger

shresthagrawal opened the pull request:
Completes #23

falafeljan commented:
i'm getting the following error:

```
(node:31335) UnhandledPromiseRejectionWarning: TypeError: this.parser.getResourceLocations is not a function
    at ParserManager.getResources (.../ink-desktop/app/background.js:520:24)
    at manageResource (.../ink-desktop/app/background.js:669:27)
```

is there probably an unpublished version of `ableton-parser`? please merge https://github.com/ununu-p2p/als-parser/pull/10 once fixed, so we can update with the correct module name (`als-parser`).

shresthagrawal commented:
> i'm getting the following error:
> 
> ```
> (node:31335) UnhandledPromiseRejectionWarning: TypeError: this.parser.getResourceLocations is not a function
>     at ParserManager.getResources (.../ink-desktop/app/background.js:520:24)
>     at manageResource (.../ink-desktop/app/background.js:669:27)
> ```
> 
> is there probably an unpublished version of `ableton-parser`? please merge [ununu-p2p/als-parser#10](https://github.com/ununu-p2p/als-parser/pull/10) once fixed, so we can update with the correct module name (`als-parser`).

Hey I have fixed that and also updated the module name here!
